### PR TITLE
v4.25.3 Fix regression to push app related metrics to StatsD

### PR DESCRIPTION
**Pull request by denis-shuvalov**
Hot fix. Fix regression to enable pushing missing metrics to Datadog. 
The STATSD registry are being added to Metrics.Registries for paid apps.
This allows to send application related metrics in micrometer mode.

### CODE REVIEW

File: tests/unit/test_micrometer_metrics.py [hunk 1]
```diff
@@ -48,17 +48,31 @@ def setUp(self) -> None:
...
+            self.assertEqual(len(metrics_registries), 1)
+
+    @patch("buildpack.telemetry.datadog.is_enabled", return_value=True)
+    def test_paidapps_metrics_registry_statsd(self, is_enabled):
+        with patch.dict(os.environ, {"PROFILE": "some-random-mx-profile"}):
+            result = metrics.configure_metrics_registry(Mock())
+            metrics_registries = result.get("Metrics.Registries")
+            self.assertEqual(
+                metrics_registries,
                 metrics.PAIDAPPS_METRICS_REGISTRY,
```
Comment (pappachino) [inline tests/unit/test_micrometer_metrics.py:69]: Won't the test for StatsD registry have the new `STATSD_REGISTRY` too?

File: buildpack/telemetry/metrics.py [hunk 2]
```diff
@@ -164,7 +171,12 @@ def configure_influx_registry(m2ee):
     if util.is_free_app():
         return {"Metrics.Registries": FREEAPPS_METRICS_REGISTRY}
 
-    return {"Metrics.Registries": PAIDAPPS_METRICS_REGISTRY}
+    paidapps_regestries = {"Metrics.Registries": PAIDAPPS_METRICS_REGISTRY}
```
[the remark this answers is not in the document]
Comment (denis-shuvalov) [reply buildpack/telemetry/metrics.py]: refactored

File: buildpack/telemetry/metrics.py [hunk 3]
```diff
@@ -50,6 +52,11 @@
     }
 ]
 
+STATSD_REGESTRY = {
```
[the remark this answers is not in the document]
Comment (denis-shuvalov) [reply buildpack/telemetry/metrics.py]: fixed

File: tests/unit/test_micrometer_metrics.py (hunk 1 above)
Comment (denis-shuvalov) [reply tests/unit/test_micrometer_metrics.py]: Discussed, test case changed

**Review by pappachino — APPROVED** [review]
Looks good!!

**Review by svanderburg — APPROVED** [review]

**Pull request merged**